pkg/gds/secrets/token_test.go

```diff
@@ -0,0 +1,24 @@
+package secrets_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/gds/secrets"
+)
+
+// Test that the CreateToken function creates a valid token of the given length.
+func TestCreateToken(t *testing.T) {
+	// Negative or zero length should return an empty string
+	require.Equal(t, "", secrets.CreateToken(-1))
+	require.Equal(t, "", secrets.CreateToken(0))
+
+	// Returned token should not contain any unexpected characters
+	token := secrets.CreateToken(20)
+	require.True(t, secrets.ValidateToken(token), "token %s contains invalid characters", token)
```

Comment:
Should we also add a sad path test where `ValidateToken` evaluates to false?

Reply:
Good call, I've added a test for `ValidateToken` so I think that case should be covered now.